### PR TITLE
Spinner/hyp 92

### PR DESCRIPTION
**Pull request by n3op2**
# What

A simple spinner component, just realised that we have forgotten to include in the storybook. Thought we did while working on HII/Deti. Please find some screenshots below

## screengrabs
Screenshot 2024-01-24 at 11 42 23 am
Screenshot 2024-01-24 at 10 30 13 am
Screenshot 2024-01-24 at 10 30 10 am

### CODE REVIEW

File: src/Animations/Spinner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  padding: auto;
+  -webkit-animation: spin 1.75s linear infinite; /* Safari */
+  animation: spin 1.75s linear infinite;
+  }
+
+  /* Safari */
+  @-webkit-keyframes spin {
+  0% { -webkit-transform: rotate(0deg); }
+  100% { -webkit-transform: rotate(360deg); }
+  }
```
Comment (mattdean-digicatapult) [inline src/Animations/Spinner/index.tsx:42]: I thought keyframes were now supported in Safari?

File: src/Animations/Spinner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+const Animation = styled('div')`
+  border: 10px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 10px solid ${(props: SpinnerProps) => props.color || '#3498db'};
+  width: ${(props: SpinnerProps) => sizeToPixel[props.size || 'small']};
+  height: ${(props: SpinnerProps) => sizeToPixel[props.size || 'small']};
+  margin: auto;
+  padding: auto;
+  -webkit-animation: spin 1.75s linear infinite; /* Safari */
```
Comment (mattdean-digicatapult) [inline src/Animations/Spinner/index.tsx:34]: I don't think this is necessary for any modern browsers

File: src/Animations/Spinner/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+  }
+`
+const Wrapper = styled.div`
+  align-self: center;
+  justify-self: center;
+`
+
+const Label = styled.h2`
```
Comment (mattdean-digicatapult) [inline src/Animations/Spinner/index.tsx:54]: this shouldn't be a h2, it's a span

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
Also missing the outer export

File: src/Animations/Spinner/index.tsx (hunk 1 above)
Comment (n3op2) [reply src/Animations/Spinner/index.tsx]: has been removed

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**